docs/reference/crds/policies.md

```diff
@@ -5,22 +5,24 @@ View more CRD samples [here](../../../config/samples/policy). The `metadata.name
 apiVersion: config.ratify.deislabs.io/v1alpha1
 kind: Policy
 metadata:
-  name: "configpolicy"
+  name: "ratify-policy"
 spec:
+  type: required. Type of the policy.
   parameters: required. Parameters specific to this policy
 ```
-There would be exactly one CR existing in the cluster. That means if users apply a new CR, the old one would be replaced.
+Ratify would only accept a policy setting `metadata.name` as `ratify-policy` since there is at most one active policy for Ratify to use. That means if users apply a new CR with different `metadata.name`, Ratify will not update the existing policy.
```

Comment:
I'm a bit confused on the behavior and scenarios here. What happens in each of these scenarios?
- User applies a CR with `configpolicy` type and metadata.name of `ratify-policy`. Then user applies a CR with `regopolicy` type and metadata.name of `ratify-policy`. Is a new CR allowed to be created for the second `regopolicy`. Which one does ratify consider as the `active` one? How does the user get informed which one is used?
- User applies a CR with `configpolicy` type and metadata.name of `ratify-policy`. Then user applies a CR with `configpolicy` type and metadata.name of `ratify-policy2`. Does the second CR just get ignored? How is the user informed which one is being used?
- User applies a CR with `configpolicy` type and metadata.name of `ratify-policy`. Then user applies a CR with `not-real` type and metadata.name of `ratify-policy`. Does the `not-real` policy CR get ignored? Is the user prompted that this is an error if they do kubectl describe on the CR?

Reply:
good questions! In v1beta1, the metadata.name must be `ratify-policy`, and spec.type must be `config-policy` or `rego-policy`. A wrong value will cause an error while applying the CR and users could check those errors in the status.